api/routes/application/routes.py

```diff
@@ -81,22 +82,25 @@ def get_key_application_data_report(self, application_id):
                 download_name="required_data.csv",
             )
         except NoResultFound as e:
-            return {"code": 404, "message": str(e)}
+            return {"code": 404, "message": str(e)}, 404
 
     def get_applications_statuses_report(
         self, round_id: Optional[str] = None, fund_id: Optional[str] = None
     ):
         try:
             return send_file(
                 export_json_to_csv(
-                    get_general_status_applications_report(round_id, fund_id)
+                    get_general_status_applications_report(
+                        round_id if round_id else None,
+                        fund_id if fund_id else None,
```

Comment:
Another way of doing this, but not necessary:

```suggestion
                        round_id or None,
                        fund_id or None,
```